mobx/lib/src/core/atom_extensions.dart

```diff
@@ -10,11 +10,13 @@ extension AtomSpyReporter on Atom {
     context.spyReport(ObservableValueSpyEvent(this,
         newValue: newValue, oldValue: oldValue, name: name));
 
+    final actionName = context.isSpyEnabled ? '${name}_set' : null;
```

Comment:
`actionName` should be set to `name` when spy is disabled

Reply:
Also @Ascenio, there are some analyser errors: https://github.com/mobxjs/mobx.dart/runs/4590667181?check_suite_focus=true

Reply:
About the analyzer errors, they were are already in mobx's master. PR #722 fixed it.

Reply:
Do you mean like this?
```suggestion
    final actionName = context.isSpyEnabled ? '${name}_set' : name;
```

Reply:
Yes